ta/pkcs11/src/processing_asymm.c

```diff
@@ -88,13 +88,22 @@ pkcs2tee_algorithm(uint32_t *tee_id, uint32_t *tee_hash_id,
 		  TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA384, TEE_ALG_SHA384 },
 		{ PKCS11_CKM_SHA512_RSA_PKCS_PSS,
 		  TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA512, TEE_ALG_SHA512 },
+#if defined(TEE_ALG_ECDSA_SHA256) && (TEE_ALG_ECDSA_SHA256 == TEE_ALG_ECDSA_P256)
+ 		{ PKCS11_CKM_ECDSA, TEE_ALG_ECDSA_SHA256, TEE_ALG_SHA256},
```

Comment:
For `PKCS11_CKM_ECDSA`, the algorithm depends on key size and `.tee_hash_id` should be 0 so that `allocate_tee_operation()` does not compte the hash.